spacy/lang/fr/stop_words.py

```diff
@@ -1,30 +1,31 @@
 STOP_WORDS = set(
     """
 a à â abord afin ah ai aie ainsi ait allaient allons
-alors anterieur anterieure anterieures apres après as assez attendu au
-aucun aucune aujourd aujourd'hui aupres auquel aura auraient aurait auront
+alors anterieur anterieure anterieures antérieur antérieure antérieures
+apres après as assez attendu au
+aucun aucune aucuns aujourd aujourd'hui aujourd’hui aupres auquel aura auraient aurait auront
```

Comment:
- added accented versions of "antérieur/e/s";
- added "aucuns", sometimes pluralized;
- added "aujourd’hui", with right single quotation mark;